package.json

```diff
@@ -99,5 +100,18 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "jest": {
+    "collectCoverageFrom": [
```

Comment:
Probably would be good idea to add similar list to **codecov** config (under _ignore_).

Reply:
I assumed, that if those files are excluded from the report inside `coverage` folder, they won't be visible on the https://codecov.io report.

Reply:
Could you double check it?